src/main/java/XbeLoader/XbeXbSymbolDatabaseAnalyzer.java

```diff
@@ -92,14 +92,20 @@ public boolean added(Program program, AddressSetView set, TaskMonitor monitor, M
 			return false;
 		}
 		String xbePath = program.getExecutablePath();
+		// HACK: Somehow GUI broke this yet headlessAnalyzer didn't...
+		// Even if remove first forward slash before drive letter cause corruption to "Executable Location" meta, not even sure why either.
```

Comment:
This is strange...

nits: Please change comment to something shorter, e.g.

```
// Ensure path does not erroneously begin with `/` before drive letter
```

Reply:
Done, plus extend a little bit for origin of cause.